examples/ssr/src/pages/login.form.js

```diff
@@ -0,0 +1,10 @@
+
+export function post(params, request) {
+	return new Response(null, {
+		status: 301,
+		headers: {
+			'Location': '/',
+			'Set-Cookie': 'user-id=1; Path=/; Max-Age=2592000'
+		}
+	});
+}
```

Comment:
No, the convention for API routes is to name them after the file extension that will be produced by the build, so for example: `todos.json.js` produces `todos.json`. But since this is a POST route and no actual filename will be created it doesn't really matter what the extension is. So I went with `form` only because this is a regular form post. It could be anything, and doesn't need to have a "subextension" at all.

Reply:
Ah interesting, thanks for clarifying!